src/validations/notifications.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export */
+import Joi from 'joi';
+
+const validateType = Joi.object().keys({
+  type: Joi.string()
+    .min(1)
+    .valid('email', 'application')
+    .label(
+      'Enter a type of notifications(email or application) to change its status'
```

Comment:
This is the label for the field not the message to be displayed when some validations are violated. If may be a user provides and empty string the program will return 
`'Enter a type of notifications(email or application) to change its status' is required` which is wrong.

Use appropriate label.